modules/nf-core/tabix/bgziptabix/main.nf

```diff
@@ -34,8 +34,8 @@ process TABIX_BGZIPTABIX {
     stub:
     def prefix = task.ext.prefix ?: "${meta.id}"
     """
-    touch ${prefix}.gz
-    touch ${prefix}.gz.tbi
+    touch ${prefix}.${input.getExtension()}.gz
```

Comment:
Do we also need to adapt this either in the modules config or the test yml/docs?

Reply:
That's for stub, we rarely use that in our tests